jubatus/core/bandit/summation_storage.cpp

```diff
@@ -44,6 +51,32 @@ void delete_arm_(summation_storage::table_t& t, const std::string& arm_id) {
     as.erase(arm_id);
   }
 }
+arm_info_map& get_arm_info_map_(summation_storage::table_t& t,
+                                const std::vector<std::string>& arm_ids,
+                                const std::string& player_id) {
+  summation_storage::table_t::iterator iter = t.find(player_id);
+  if (iter != t.end()) {
+    return iter->second;
+  }
+  arm_info_map& as = t[player_id];
+  const arm_info a0 = {0, 0.0};
+  for (size_t i = 0; i < arm_ids.size(); ++i) {
+    as.insert(std::make_pair(arm_ids[i], a0));
+  }
+  return as;
+}
+arm_info& get_arm_info_(summation_storage::table_t& t,
+                        const std::vector<std::string>& arm_ids,
+                        const std::string& player_id,
+                        const std::string& arm_id) {
+  arm_info_map& as = get_arm_info_map_(t, arm_ids, player_id);
+  arm_info_map::iterator iter = as.find(arm_id);
+  if (iter == as.end()) {
+    throw JUBATUS_EXCEPTION(common::exception::runtime_error(
+        "arm_id is not registered"));
```

Comment:
How about adding ``arm_id`` to this message like the following? I think it is easy to debug.
```
arm_id is not registered: arm_001
```